Specs/.eslintrc.json

```diff
@@ -7,6 +7,7 @@
         "defineSuite": true
     },
     "rules": {
-        "quotes": "off"
+        "quotes": "off",
```

Comment:
I noticed this while doing the updates, I'm going to open a new PR once this one is merged to fix all of the double quotes in our tests.